### ISSUE TITLE
Change init_param: feed a disparity map initialization

### ISSUE THREAD

**Issue by loryruta** [jiaxiZeng/Parameterized-Cost-Volume-for-Stereo-Matching#6]
PCVNet is a good tradeoff between accuracy and speed, thanks again for your work.

However, due to my image size it doesn't fit my real-time requirements.
This is why I thought that initializing the model with a `disparity map` guess would allow me to use less iterations to obtain a prediction of roughly the same quality in less time.

In my application I am able to produce such rough `disparity map` candidate.

I've seen PCVNet allows a `init_param` input, a dictionary with `mu`, `sigma`, `w` describing the initial disparity space distribution.
However, setting anything different from initial arguments (and using the [provided checkpoint](https://github.com/jiaxiZeng/Parameterized-Cost-Volume-for-Stereo-Matching?tab=readme-ov-file#evaluate)), doesn't work.

Even changing the `init_mu`, `init_sigma` and `init_w` to anything different than resp. [0, 64, 128, 192], 32, and 1/num_gauss doesn't work.

Is the model/checkpoint sticked to the parameters that were used for training?

What do you suggest, otherwise, for improving the model's inference? (downsampling is not an option)

**Comment by jiaxiZeng** [comment]
Thank you for your interest in our work. The provided checkpoint (ckpt) only supports initializing the Gaussian distribution from the default Gaussian parameters. If you wish to use multi-scale inference, you would need to train or fine-tune the model with multi-scale (or cascade) strategy to allow the model to adapt to initialization from the previous scale.

If you have real-time requirements and are using video streams as input, you may want to check out our new work, **TC-Stereo**.

**Comment by loryruta** [comment]
Thanks for the quick answer. No, I'm working with videos and thus I don't need additional features such as temporal consistency. My application is what PCVNet was done for (`(left, right) -> disparity`).

About training/fine-tuning, on which hardware did you train PCVNet? I would like to understand the minimal requirements... It's not really clear from the paper; maybe?
> All of the experiments except for the ablation study are conducted on 2 NVIDIA GeForce RTX 3090 GPUs

**Comment by jiaxiZeng** [comment]
It depends on your image size during training, typically, two RTX3090 are enough.
